app/controllers/concerns/account_concern.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module AccountConcern
+  extend ActiveSupport::Concern
+
+  ACCOUNT_SESSION_REQUEST_HEADER_NAME = "HTTP_GOVUK_ACCOUNT_SESSION"
+  ACCOUNT_SESSION_RESPONSE_HEADER_NAME = "GOVUK-Account-Session"
+  ACCOUNT_END_SESSION_RESPONSE_HEADER_NAME = "HTTP_GOVUK_ACCOUNT_END_SESSION"
+  ACCOUNT_SESSION_DEV_COOKIE_NAME = "govuk_account_session"
+
+  included do
+    before_action :fetch_account_session_header
+    before_action :set_account_session_header
```

Comment:
I don't think we need the `before_action :set_account_session_header` any more

Reply:
removed